Mathlib/MeasureTheory/Integral/IntegrableOn.lean

```diff
@@ -81,6 +81,13 @@ theorem hasFiniteIntegral_restrict_of_bounded [NormedAddCommGroup E] {f : α →
 variable [NormedAddCommGroup E] {f g : α → ε} {s t : Set α} {μ ν : Measure α}
   [TopologicalSpace ε] [ContinuousENorm ε]
 
+theorem hasFiniteIntegral_restrict_of_bounded_enorm {f : α → ε} {s : Set α} {μ : Measure α}
```

Comment:
```suggestion
theorem HasFiniteIntegral.restrict_of_bounded_enorm {f : α → ε} {s : Set α} {μ : Measure α}
```
for anon dot not if you wish

Reply:
Sure; renamed this one (and its non-enorm analogues).